### PR TITLE
Add bottleneck within photography ads and enable profiler

### PR DESCRIPTION
**Pull request by breedx-splk**
So if ads are requested for the `photography` context (category) then they will be slow. On my mac it's about 5.5s.

This is controlled by an environment var named `ENABLE_COPYRIGHT_CERTIFICATION`, which is disabled by default. However, the Dockerfile here has been changed to turn this _on_ by default.

I have confirmed that the Dockerfile can produce an image that contains the bottleneck.
This may not be complete -- may require collector endpoint to be changed/specified somewhere.

### CODE REVIEW

File: src/adservice/build.gradle [hunk 1]
```diff
@@ -88,13 +99,6 @@ task downloadRepos(type: Copy) {
     into offlineCompile
 }
 
-task adService(type: CreateStartScripts) {
-    mainClassName = 'hipstershop.AdService'
-    applicationName = 'AdService'
-    outputDir = new File(project.buildDir, 'tmp')
-    classpath = startScripts.classpath
-}
-
```
Comment (dmitryax) [inline src/adservice/build.gradle:97]: Why is this removed?

Comment (breedx-splk) [reply src/adservice/build.gradle:97]: It wreaks havoc with the gradle plugin that expects to install the java agent via "normal" means. The previous version of the gradle build script had the topmost start scripts disabled and then re-patched in via this custom mechanism. Gradle doesn't do well with overcustomization. :)

Comment (dmitryax) [reply src/adservice/build.gradle:97]: Ok, sounds good. Can we remove the usage instead of commenting it then?

**Review by dmitryax — APPROVED** [review]
LGTM

File: src/adservice/Dockerfile [hunk 2]
```diff
@@ -22,5 +22,8 @@ RUN GRPC_HEALTH_PROBE_VERSION=v0.3.1 && \
 WORKDIR /app
 COPY --from=builder /app .
 
+# Allow bottlenecks via copyright certification
+ENV ENABLE_COPYRIGHT_CERTIFICATION=true
+
 EXPOSE 9555
-ENTRYPOINT ["/app/build/install/hipstershop/bin/AdService"]
```
Comment (dmitryax) [inline src/adservice/Dockerfile:26]: Curious how the path appeared to be changed?

Comment (breedx-splk) [reply src/adservice/Dockerfile:26]: Yeah, so the custom gradle (java build tool) magic that was there before was setting it up in a custom path. It's using a more standard gradle default now.

Comment (dmitryax) [reply src/adservice/Dockerfile:26]: Thanks for clarification!

**Review by dmitryax — APPROVED** [review]
LGTM. Disregard the CI tests, I'll fix them later

Feel free to merge

**Comment by breedx-splk** [comment]
Going to close since we don't want to merge this, but I'll leave the branch around. Thanks!
